### PR TITLE
Fix "from source" installation method

### PR DESCRIPTION
**Pull request by smondet**
There should be at least a link on that page to get `ocaml-4.01.0.tar.gz`.

I removed “This is unlikely the method you want to use unless you are a developer of the OCaml compiler itself or trying to install on some esoteric platforms.” because there are many cases when one would prefer getting the original sources.

### CODE REVIEW

File: site/docs/install.md [hunk 1]
```diff
@@ -537,7 +537,7 @@ See [here](https://github.com/hcarty/ocamlbrew).
 
 
 ## From Source
-Download the [latest development version](/releases/svn.html) using
-subversion and follow the instructions included therein. This is
-unlikely the method you want to use unless you are a developer of the
-OCaml compiler itself or trying to install on some esoteric platforms.
+
+Download the [source of the latest version](http://caml.inria.fr/ocaml/release.en.html)
```
Comment (Chris00) [inline site/docs/install.md]: The link should be `/releases/4.01.0.html`.  However, one may not want to have to update it manually (one likely will forget).  So a script checking the latest version available in `/releases/` would be good.  Or else, change the text and simply point to `/releases`.

Comment (smondet) [inline site/docs/install.md]: Oh I did not know there was http://ocaml.org/releases/ (there is no link from anywhere else, right?).

So this line should be

```
Download the [source of the release you want](/releases/)
```

?

should I add a commit or do you prefer a “rebase”?

Comment (Chris00) [inline site/docs/install.md]: On Tue, 4 Mar 2014 15:03:35 -0800, Sebastien Mondet wrote:

> Oh I did not know there was http://ocaml.org/releases/ (there is no
> link from anywhere else, right?).

Indeed, I am afraid so.

> So this line should be
> 
> Download the [source of the release you want](/releases/)

Something like that — maybe a bit better (that does not sound too
engaging to me).

> should I add a commit or do you prefer a “rebase”?

I prefer you rebase.

**Comment by smondet** [comment]
I've rebased the PR.

**Comment by Chris00** [comment]
Merged as db5e4ad

**Pull request closed without merging**